__tests__/app.test.js

```diff
@@ -70,3 +70,57 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:articleid", () => {
+  test("responds with status code 200", () => {
+    return request(app).get("/api/articles/6").expect(200);
+  });
+  test("responds with an article object", () => {
+    return request(app)
+      .get("/api/articles/5")
+      .then(({ body }) => {
+        expect(typeof body).toBe("object");
+      });
```

Comment:
this is a bit of a redundant test, knowing you're going to have to check against the object properties later it doesn't add a different behaviour

Reply:
removed this test